cert-manager/smaug/kustomization.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
add the following field: 
```yaml
namespace: openshift-ingress
```
And remove this from `certificate.yaml` and `issuer.yaml`